examples/tutorials/asr_inference_with_ctc_decoder_tutorial.py

```diff
@@ -161,26 +163,27 @@
 # Construct Beam Search Decoder
 # -----------------------------
 #
-# The decoder can be constructed using the
-# :py:func:`torchaudio.prototype.ctc_decoder.kenlm_lexicon_decoder`
-# factory function.
-# In addition to the previously mentioned components, it also takes in
-# various beam search decoding parameters and token/word parameters.
+# The decoder can be constructed using the ``kenlm_lexicon_decoder``
+# factory function from ``torchaudio.prototype.ctc_decoder``. In addition
+# to the previously mentioned components, it also takes in various beam
+# search decoding parameters and token/word parameters. The full list of
+# parameters can be found
+# :py:func:`here <torchaudio.prototype.ctc_decoder.kenlm_lexicon_decoder>`.
```

Comment:
nit: I think instead of waiting to provided the link to the document at the end end of section (`The full list of parameters ...`), we can link it in the first mention of the function.
(But you might have chosen the current working to work around the formatter warning.) 

```
# The decoder can be constructed using the :py:func:`torchaudio.prototype.ctc_decoder.kenlm_lexicon_decoder`.
# In addition to the previously mentioned components, it also takes in various beam
# search decoding parameters and token/word parameters.
```